hw5/tests/test_hw5_t01.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+
+from hw5.hw5_t01 import Homework, Student, Teacher
+
+
+def test_attributes_of_teacher_class():
+    teacher = Teacher("Smith", "Adam")
+    assert teacher.last_name, teacher.first_name == ("Smith", "Adam")
+
+
+def test_attributes_of_student_class():
+    student = Student("Doe", "John")
+    assert student.last_name, student.first_name == ("Doe", "John")
+
+
+def test_attributes_of_homework_class():
+    homework = Homework("Tasks", 3)
+    assert homework.text, homework.deadline == ("Tasks", 3)
+
+
+def test_negative_is_active_method_of_homework_class():
```

Comment:
negative is active method of homeworkclass, isn't it?

`test_expaired_homework_is_not_active`

Reply:
No pun intended XD